_data/contributors.yml

```diff
@@ -49,11 +49,22 @@
 
 - name: Vaibhav Thakkar
   photo: Vaibhav.jpg
-  info: "Google Summer of Code 2023 Contributor"
+  info: "Research Intern at CERN, Google Summer of Code 2023 Contributor"
   email: vaibhav.thakkar.22.12.99@gmail.com
   education: "Electrical Engineering and Computer Science, Indian Institute of Technology, Kanpur, India"
   active: 1
   projects:
+    - title: "Add support for pointers in reverse mode AD in Clad"
+      status: Ongoing
+      description: |
+        Clad is an automatic differentiation library based on Clang which provides the
+        necessary facilities for code transformation.
+        Currently, clad has limited support for differentiation w.r.t pointers, and
+        that too only in forward mode. This project aims to add support for differentiation w.r.t
+        pointers in reverse mode AD in clad, allowing the user to combine the power of
+        reverse mode AD with the flexibility of pointers in C++.
```

Comment:
Maybe we can take some inspiration in terms of description from here https://compiler-research.org/open_projects#add-support-for-differentiating-with-respect-to-multidimensional-arrays-or-pointers-in-clad

Reply:
updated.